azure_arc_servers_jumpstart/docs/arc_azuresentinel.md

```diff
@@ -0,0 +1,101 @@
+
+    ![](../img/azuresentinel/03.png)
+
+* To deploy the ARM template, navigate to the [deployment folder](../azuresentinel/arm) and run the below command:
+
+  ```bash
+    az deployment group create --resource-group <Name of the Azure Resource Group> \
+    --template-file <The *sentinel-template.json* template file location> \
+    --parameters <The *sentinel-template.parameters.json* template file location>
+  ```
+
+# Azure Arc enabled VMs onboarding on Azure Sentinel 
+
+Once you have deployed Azure Sentinel on your Log Analytics workspace, you need to start connecting data sources. There are connectors for Microsoft services, 3rd party solutions of the security ecosystem or you can also use Common Event Format (CEF), Syslog or REST-API to connect your data sources with Azure Sentinel. For servers and VMs, you can install the MMA or Sentinel agent that collects the logs and forwards them to Azure Sentinel. You can deploy the agent in multiple ways leveraging Azure Arc: 
+
+* Using the **[Extension Management](arc_vm_extension_mma_arm.md)** feature in Azure Arc for servers that enables you to deploy the Microsoft Monitoring Agent VM extensions to non-Azure Windows and Linux VMs. You can use the Azure Portal, Azure CLI, an ARM template or PowerShell script to manage the extension deployment to Arc servers, both Linux and Windows. 
+
+* Setting up **[Azure Policies](arc_policies_mma.md)**, with this approach you will assign a policy to audit if the Azure Arc enabled machine has the Microsoft Monitoring Agent (MMA) installed, if not, you will use the extensions feature to automatically deploy it to the VM with a remediation task, an enrollment experience that levels to Azure VMs. 
+
+# Clean up environment
+
+Complete the following steps to clean up your environment.
+
+* Remove the virtual machines from each environment by following the teardown instructions from each guide.
+
+    - *[GCP Ubuntu VM](gcp_terraform_ubuntu.md) / [GCP Windows VM](gcp_terraform_windows.md)*
+    - *[AWS Ubuntu VM](aws_terraform_ubuntu.md)*
+    - *[VMware Ubuntu VM](vmware_terraform_ubuntu.md) / [VMware Windows Server VM](vmware_terraform_winsrv.md)*
+    - *[Local Ubuntu VM](local_vagrant_ubuntu.md) / [Local Windows VM](local_vagrant_windows.md)*
+
+* Remove the Log Analytics workspace by executing the following script in AZ CLI. Provide the workspace name you used when creating the Log Analytics Workspace.
+
+    ```bash
+    az monitor log-analytics workspace delete --resource-group <Name of the Azure Resource Group> --workspace-name <Log Analytics Workspace Name> --yes
+    ```
```

Comment:
Incl. screenshot

Reply:
command does not have an output, should I still include it?

Reply:
never mind, it's fine